src/js/edu-benefits/5490/config/form.js

```diff
@@ -518,11 +518,21 @@ const formConfig = {
             },
             'view:hasTrainings': {
               'ui:title': 'Do you have any training after high school?',
-              'ui:widget': 'yesNo'
+              'ui:widget': 'yesNo',
+              'ui:options': {
+                hideIf: form => {
+                  const status = _.get('highSchool.status', form);
+                  return status === 'discontinued' || status === 'graduationExpected' || status === 'neverAttended';
+                }
+              }
             },
             postHighSchoolTrainings: _.merge(postHighSchoolTrainingsUi, {
               'ui:options': {
-                expandUnder: 'view:hasTrainings'
+                expandUnder: 'view:hasTrainings',
+                hideIf: form => {
```

Comment:
Hmm, ideally you wouldn't need to do this. I'm going to add something about it to our expandUnder issue (https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2046)